jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-2a.yaml

```diff
@@ -266,7 +266,7 @@
 
             cat > /tmp/sbd_prepare_$admin <<EOSCRIPT
               # preparation of iSCSI
-              zypper --gpg-auto-import-keys -p http://download.opensuse.org/repositories/devel:/languages:/python/SLE_12_SP2/ --non-interactive install python-sh
+              zypper --gpg-auto-import-keys -p http://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_12_SP4/ --non-interactive install python-sh
```

Comment:
There are also 12-SP2 and 3 repos - or are you only using this for SOC9 ?

Reply:
For a small python-sh it does not make sense to distinguish between SP. The package from SP4 repo also works in SP2 in my test environment.